src/ViewInfo.h

```diff
@@ -101,6 +101,7 @@ class AUDACITY_DLL_API NotifyingSelectedRegion : public wxEvtHandler
 // See big pictorial comment in TrackPanel.cpp for explanation of these numbers
 enum : int {
    // constants related to y coordinates in the track panel
+   kAffordancesAreaHeight = 20,
```

Comment:
This constant is only used in TrackPanel.cpp -- perhaps it should be defined only in that file, until need arises to reuse the constant elsewhere.

Reply:
The comment above says: constants related to y coordinates in the track panel
And this is not the only constant used only in TrackPanel.cpp, I'd preffer to keep it here to group related things together